.gitignore

```diff
@@ -0,0 +1,7 @@
+test_s3.log
```

Comment:
looks like these files are for test purpose and shouldn't be listed here

Reply:
fixed